packages/typespec-vscode/src/vscode-cmd/create-tsp-project.ts

```diff
@@ -195,71 +159,131 @@ export async function createTypeSpecProject(
         return;
       }
 
+      const defaultLibraries =
```

Comment:
i thikn you can just remove this setting, `scaffoldNewProject` should just use teh values from the templates by defautl already

Reply:
could you help to double confirm? i think these default values are not handled by ```scaffoldNewProject()``` directly but  set by ```makeScaffoldingConfig()``` in ```tsp init``` logic before calling ```scaffoldNewProject()```.

Reply:
oh you are righ, should we maybe export this functio instead then as well?